modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -53,15 +53,13 @@ def __init__(self, data, length=None, width=None, ip=None, call_queue=None):
         self._width_cache = width
         self._ip_cache = ip
         self._identity = uuid.uuid4().hex
-
         logger = get_logger()
         logger.debug(
-            "Partition ID: {}, Height: {}, Width: {}, Node IP: {}".format(
-                self._identity,
-                str(self._length_cache),
-                str(self._width_cache),
-                str(self._ip_cache),
-            )
+            "Partition ID: %s, Height: %s, Width: %s, Node IP: %s",
+            self._identity,
+            str(self._length_cache),
+            str(self._width_cache),
+            str(self._ip_cache),
```

Comment:
```suggestion
            self._length_cache,
            self._width_cache,
            self._ip_cache,
```
no need to explicitly `str()` things, as `%s` formatter calls that for us